Gemfile

```diff
@@ -30,6 +30,7 @@ end
 
 group :test do
   gem 'webrick'
+  gem 'factory_girl', '~> 4.9'
```

Comment:
Curious about this, `factory_girl` was renamed a while back to `factory_bot` and shouldn't the following line for `factory_bot_rails` satisfy the app's needs?